src/content/docs/de/about/book.md

```diff
@@ -0,0 +1,32 @@
+---
+title: The Tauri Book
+i18nReady: true
+---
+
+:::note[Fortschrittsbericht]
+
+Wir arbeiten aktiv an der Erstellung und dem Schreiben von "The Tauri Book". Aufgrund des enormen Wachstums von Tauri sind wir in Verzug geraten, haben aber kürzlich das Projekt neu priorisiert. Wir haben derzeit noch keine Informationen über die Zeitpläne einer Veröffentlichung, aber Sie können diese Seite im Auge behalten, um über Updates informiert zu werden.
+
+Wir entschuldigen uns für die Verzögerung, die über das ursprünglich kommunizierte Veröffentlichungsdatum hinausgeht. Wenn Sie über GitHub Sponsors oder Open Collective gespendet haben und einen Rückerstattungsantrag stellen möchten, können Sie dies über Open Collective: [Kontaktieren Sie Tauri auf Open Collective](https://opencollective.com/tauri/contact).
+
+:::
+
+### Überblick
+
+"The Tauri Book" führt Sie durch die Geschichte von Tauri und die Designentscheidungen, die wir getroffen haben. Es behandelt auch ausführlich, warum Datenschutz, Sicherheit und Nachhaltigkeit wichtig und grundlegende Aspekte sind, die Sie auf jedes moderne Softwareprojekt anwenden können.
+
+Die folgenden Themen werden behandelt:
+
+- Die Methodik und die Gründe für das Design von Tauri
+- Die Möglichkeiten, die Sie haben, wenn Sie mit Tauri entwickeln
+- Dass Sie nicht zwischen schneller Lieferung und Nachhaltigkeit und Verantwortlichkeit wählen müssen
+- Warum wir die Rust-Sprache als Bindung und Anwendungsschicht für Tauri ausgewählt haben
+- Warum eine Binärprüfung wichtig ist
```

Comment:
That whole block sounds weird tbh. And the last point is somewhat of an incorrect translation.

Reply:
I look in to the original later. But Yes, i now see the Problem i think. Some Words shouldn't be Translated though imo.